lib/koans/08_structs.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Structs do
+  use Koans
+
+  defmodule Person do
+    defstruct [:name, :age ]
+  end
+
+  koan "structs are defined and named after a module" do
+    assert %Person{}
```

Comment:
Yeah, I wanted some way to show how to create structs without going to deep. I can remove that koan.